src/__test__/test-helper.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+const shouldMatchSnapshot = (Name, Component) => {
+  describe(`"${Name}"`, () => {
```

Comment:
This describe block, and by extension the `Name` parameter, isn't needed in this function. We should wrap this function in a describe block of the calling tests.